app/src/app/[lng]/[inventory]/data/manage-sectors/SectorTabs.tsx

```diff
@@ -0,0 +1,652 @@
+"use client";
+import { InventoryProgressResponse } from "@/util/types";
+import {
+  Box,
+  Card,
+  CheckboxCard,
+  createListCollection,
+  Field,
+  Icon,
+  Input,
+  ProgressCircle,
+  Tabs,
+  Text,
+  Textarea,
+} from "@chakra-ui/react";
+import { SerializedError } from "@reduxjs/toolkit";
+import { FetchBaseQueryError } from "@reduxjs/toolkit/query";
+import { TFunction } from "i18next";
+import React, { FC, useEffect } from "react";
+import { SubSectorWithRelations } from "../[step]/types";
+import { StationaryEnergyIcon } from "@/components/icons";
+import { BiSelectMultiple } from "react-icons/bi";
+
+import { MdInfoOutline, MdRemoveFromQueue } from "react-icons/md";
+import { RiErrorWarningFill } from "react-icons/ri";
+import { CgRemoveR } from "react-icons/cg";
+
+import {
+  SelectContent,
+  SelectItem,
+  SelectLabel,
+  SelectRoot,
+  SelectTrigger,
+  SelectValueText,
+} from "@/components/ui/select";
+import { Button } from "@/components/ui/button";
+import { api } from "@/services/api";
+import { toaster } from "@/components/ui/toaster";
+import RouteChangeDialog from "./RouteChangeDialog";
+import { usePathname, useRouter } from "next/navigation";
+import ProgressLoader from "./ProgressLoader";
+
+interface SectorTabsProps {
+  t: TFunction;
+  inventoryData: InventoryProgressResponse | undefined;
+  isInventoryDataLoading: boolean;
+  inventoryDataError: FetchBaseQueryError | SerializedError | undefined;
+}
+
+interface QuickActionInputs {
+  notationKey: string;
+  explanation: string;
+}
+
+interface CardInputs {
+  notationKey: string;
+  explanation: string;
+}
+
+const SectorTabs: FC<SectorTabsProps> = ({
+  inventoryData,
+  inventoryDataError,
+  isInventoryDataLoading,
+  t,
+}) => {
+  const router = useRouter();
+
+  const [unfinishedSubsectorsData, setUnfinishedSubsectorsData] =
+    React.useState<SubSectorWithRelations[] | undefined>([]);
+  // State to track selected subsector IDs per sector (keyed by sector ID)
+  const [selectedCardsBySector, setSelectedCardsBySector] = React.useState<
```

Comment:
Can you import useState directly, since it's used in so many places in this file?